build.gradle

```diff
@@ -428,7 +428,7 @@ task buildDependencyReport(type: Exec) {
   outputs.file("${buildDir}/distributions/dependencies-${version}.csv")
   environment makeEnvironment
   commandLine shell
-  args '-c', "source ./set_env.sh && 3rd_party/dependency_report.sh --csv \"${outputs.files.singleFile}\""
+  args shell_flag, "${set_env} && ${cmake} -D OUTPUT_FILE=\"${outputs.files.singleFile}\" -P 3rd_party/dependency_report.cmake"
```

Comment:
TODO: change variables here to camel case once #2347 has been merged to main

Reply:
You should be able to merge main into this PR branch now to get the updates